internal/deploys/models.go

```diff
@@ -51,29 +52,13 @@ func (db dataStore) createNew(environmentID, serviceID int) (ServiceInstance, er
 		return ServiceInstance{}, fmt.Errorf("error persisting service instance: %v", err)
 	}
 
-	// retrieve the same service instance record back from the db but now with all the
-	// associations populated.
-	err := db.Preload("Service").
-		Preload("Environment").
-		First(&newServiceInstance, newServiceInstance.ID).Error
-
-	return newServiceInstance, err
+	return newServiceInstance, nil
 }
 
-func (db dataStore) getByEnvironmentAndServiceName(environmentName, serviceName string) (ServiceInstance, error) {
+func (db dataStore) getByEnvironmentAndServiceID(environmentID, serviceID int) (ServiceInstance, error) {
```

Comment:
what's your thoughts on requiring comments on all methods. I noticed we do it pretty consistently in Controllers, but not in our models, I'm of the opinion of removing the decision of whether to have a comment in favor of always having one (at the understandable cost of more chatty code).